docs/code-examples/image_simple.py

```diff
@@ -0,0 +1,16 @@
+"""Create and log an image."""
+
+import numpy as np
+import rerun as rr
+from PIL import Image, ImageDraw
+
+# Create an image with Pillow
+image = Image.new("RGB", (300, 200), color=(255, 0, 0))
+draw = ImageDraw.Draw(image)
+draw.rounded_rectangle((50, 50, 150, 150), fill=(0, 255, 0), radius=20)
```

Comment:
If possible I'd prefer if we didn't need Pillow (has to be installed) for the simple example. Numpy is a dependency of rerun and we should be able to draw a (non rounded) rectangle by just using array slicing with pure numpy

Reply:
OK, done. For image_advanced I'd keep using Pillow as that's what many users will want to use anyways (the rest being cv2 likely). Changed depth_image and segmentation image as well.